src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -207,6 +209,125 @@ const Activity = React.createClass({
   },
 });
 
+const EPH = React.createClass({
```

Comment:
Is this "events per hour"?

Should rename this `EventsPerHour` and be explicit.